src/Raven.Server/Documents/Indexes/IndexStore.cs

```diff
@@ -1002,15 +1003,25 @@ public IEnumerable<Index> GetIndexes()
 
         public void RunIdleOperations()
         {
-            AsyncHelpers.RunSync(HandleUnusedAutoIndexes);
-            AsyncHelpers.RunSync(DeleteSurpassedAutoIndexes);
+            DatabaseRecord record;
+            using (_serverStore.ContextPool.AllocateOperationContext(out TransactionOperationContext context))
+            using (context.OpenReadTransaction())
+                record = _serverStore.Cluster.ReadDatabase(context, _documentDatabase.Name);
+
+            if (record.Topology.Members[0] != _serverStore.NodeTag)
```

Comment:
@ayende Is there a possibility to have `null` here or 0 members? I need to apply similar fix to v4.0

Reply:
No, it can't be null / empty